### ISSUE TITLE
"Requests to the server have been blocked by an extension", showing after adding filter

### ISSUE THREAD

**Issue by TheGift73** [gorhill/uBlock#1330]
[Regarding this issue](https://github.com/gorhill/uBlock/issues/1329) which is now resolved.

I added `iframe.4bc.co/split.html` to My Filters in the Options area for uBlock Origin. (latest stable version of Chrome running on Windows 7 Ultimate)

**[update]** Just noticed that I was asked to actually add the filter `4bc.co/split-top.html`, but the result is the same.

I used the default settings for the extension with the exception of adding the filter suggested by @IDKwhattoputhere here. It doesn't make the article unreadable and am quite happy with the filters results as I am no longer redirected to the 4bc.co site after the advert that was causing the initial issue.

Screenshot showing more information can be [seen here](http://imgur.com/8XHNStn). Image also attached (please see update above)
![req](https://cloud.githubusercontent.com/assets/191014/12735462/109a1770-c940-11e5-9227-a52c51b099e9.jpg)

[Additional screenshot](http://imgur.com/LUiPeTV) added showing My Filters to help
![Uploading yyy.jpg…]()

I am posting as requested by @gorhill

Regards,

Richard

**Comment by IDKwhattoputhere** [comment]
Looks exactly the same on my machine. I always thought it was supposed to look like that.

**Comment by gorhill** [comment]
With FF, frames for which resource fetching failed are left blank. However with Chromium they are left with that message. Normally uBO properly hide `iframe` elements, however here we are dealing with a `frame` element, and it needs a slightly different treatment than an `iframe` element.

**Comment by gorhill** [comment]
Ok, I had a fix for this, however after reading more about [frameset](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/frameset)/[frame](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/frame), I decided this will be a `wontfix`:

- It's only a cosmetic issue;
- That cosmetic issue affects only Chromium-based browsers;
- `frameset`/`frame` are deprecated: "This feature has been removed from the Web standards".

Not worth to add code for this.

**Comment by TheGift73** [comment]
Cool. Thank you for looking into it more and replying.
